pipeline_steps/kibana.groovy

```diff
@@ -62,7 +62,9 @@ def kibana_tests(vm=null){
         export PATH=\$PATH:./phantomjs-2.1.1-linux-x86_64/bin
         # Remove any existing screenshots from old runs
         rm -f *.png
-        python conf-gen.py --secure --password-file /etc/openstack_deploy/user_rpco_secrets.yml \
+
+        export PASSWORD=\$(grep -Ir kibana_password /etc/openstack_deploy/ | head -1 | sed 's/.*\\: //')
```

Comment:
The upgrade script creates `user_rpco_secrets.yml` based on `user_extras_secrets.yml` as part of the upgrade process, but doesn't delete `user_extras_secrets.yml` afterwards. @cjloader suggested that it might be left as a backup.

https://github.com/rcbops/rpc-openstack/blob/mitaka-13.1/scripts/test-upgrade.sh#L49
https://github.com/rcbops/rpc-openstack/blob/mitaka-13.1/scripts/update-secrets.sh

I'm going to change the `head -1` to `tail -1`, though, for now in case we want to stay with this method.